### PR TITLE
Updates to query parameters documentation

### PR DESCRIPTION
**Pull request by susodapop**
Straightforward edits. Added a warning about SQL injection. I'm not certain if the documentation is accurate when it says that use of _any query parameter_ requires Full Access to the database (see line 49). I'd like to remove that if it isn't true.

### CODE REVIEW

File: website/_kb/user-guide/querying/query-parameters.md [hunk 1]
```diff
@@ -10,46 +10,68 @@ title: Query Parameters
...
-in embeds or shared dashboards. Also, parameters require Full Access
-permission to the data source (vs. View Only).
+
+Query Parameters only work within Redash and are not supported in embeds or shared dashboards. Also, a Redash user must have [Full Access]({% link _kb/user-guide/users/permissions-groups.md  %}) permission to the data source to use Query Parameters. 
+
 {% endcallout %}
 
+### Date Parameters
+
+In many DBAPI's, you can use a string value to represent a date/time which is equivalent to a Text parameter in Redash. Redash's Date parameters are preferable because they have a special calendar interface to make choosing a date easy. If you know in advance, however, that your date parameters will always come from a short list, you can instead use the `Dropdown List` or `Query Based Dropdown List` type.
```
Comment (arikfr) [inline website/_kb/user-guide/querying/query-parameters.md]: "In many DBAPI's" -> "With many databases,"

**Comment by susodapop** [comment]
Updated based on your feedback. I added my first PNG to the docs -- it didn't render properly on my development server but I think it should work on Netlify.

**Comment by arikfr** [comment]
> I added my first PNG to the docs -- it didn't render properly on my development server but I think it should work on Netlify.

In what way it didn't render properly?

**Comment by arikfr** [comment]
Looks good 👍 

Let's remove the unrelated change in `getting-started.md` and merge this?

**Pull request merged**